models.js

```diff
@@ -169,32 +218,50 @@ Statistiky:
 
     if (!isHit) {
       this.combat.tellAll(
-        `Hrac ${this.name} netrafil (${roll} + ${this.attack} + ${this.buff} / ${target} (10 + ${amount} + ${targetDef}))`
+        this.game.getTranslation("server.player.combat.didnt_hit", {
```

Comment:
This would be better with just one interpolated value, something like `debug` which would contain the final string that should show up in between `()`. Then, when calculation changes, translation does not need to change.

Reply:
You mean something like this?

```js

const calculation = `${roll} + ${this.attack} + ${this.buff} / ${target} (10 + ${amount} + ${targetDef}`
const translation = this.game.getTranslation(
    "server.player.combat.didnt_hit",
    { calculation: calculation }
)

this.combat.tellAll(translation)
...
```

Reply:
Yes, that way, if calculation changes, you don't need to change the translation itself.